doc/modeling/indices.md

```diff
@@ -0,0 +1,42 @@
+# Indices
+
+## PostgreSQL covering indices (INCLUDE)
```

Comment:
I've mostly seen the "indexes" version written out. Any reason to use indices here?

Reply:
My grammar-obsessive Latin side is what makes me use indices. It's true that indexes is more common, will change it everywhere.